src/runtime-rs/crates/hypervisor/src/kernel_param.rs

```diff
@@ -28,6 +29,18 @@ impl Param {
             value: value.to_owned(),
         }
     }
+
+    pub fn to_string(&self) -> Result<String> {
+        if self.key.is_empty() && self.value.is_empty() {
+            Err(anyhow!("Empty key and value"))
+        } else if self.key.is_empty() {
+            Err(anyhow!("Empty key"))
+        } else if self.value.is_empty() {
```

Comment:
Is an empty value key-value valid? For example `a=b ccc`, `ccc` is an option without an value.

Reply:
Yes, an value w/o a key is valid.
e.g. `agent.debug_console` is a standalone flag.

Reply:
A key w/o a value is valid. It works as a standalone flag passed to kernel parameters. e.g. `agent.debug_console` is one of them.